server/issue.go

```diff
@@ -363,32 +367,51 @@ func httpAPIGetSearchIssues(ji Instance, w http.ResponseWriter, r *http.Request)
 		return http.StatusInternalServerError, err
 	}
 
-	jqlString := r.FormValue("jql")
-
-	searchRes, err := client.SearchIssues(jqlString, &jira.SearchOptions{
-		MaxResults: 50,
-		Fields:     []string{"key", "summary"},
-	})
-	if err != nil {
-		return http.StatusInternalServerError,
-			errors.WithMessage(err, "failed to get search results")
+	q := r.FormValue("q")
+	var exact *jira.Issue
```

Comment:
nit: Propose to change `exact` to `exactIssue` or something in the variable name that recognizes `exact` is of type `*jira.Issue`

Reply:
That'd be too Java...

Reply:
Noted that Java naming convention is bad! 😆